### PR TITLE
Lesson 5 comments 1

### PR DESCRIPTION
**Pull request by canyon289**
Code wise this is looking really solid! Feel free to merge this pr, or close it directly and incorporate comments on your own branch. Whatever works for you

Left comments largely on narration and adding a bit more explanatory elements. At the start I added a bunch of stylistic comments but realized I should focus on code at higher level.

Biggest comments
* Lead in with more real world examples of categorical regression but then moving onto crepes
* Highlight where ideas span across data science, for instance softmax is used in SVMS and Neural Nets etc
  * Adding these to lesson references would be great
* Theres a few areas where you can call back to @tomicapretto's lesson, and steal his graphics on identifiability and such

I think this is largely good to start getting slidified with correct headers and such

### CODE REVIEW

File: lesson_05.ipynb [hunk 1]
```diff
@@ -72,6 +72,28 @@
     "Sometimes, you just wanna eat something. And when you travel a lot, like I do, sometimes that craving can be directed at a _good_ crêpe. But a really good one you, know just a touristic trap."
```
Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Love that, will include </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Got it. Note that the "hide-code" plugin didn't work well for me for the Intro course. We'll see if it works better now...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Sure, I can add that point here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Got it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Yes, I'm planning to add explanation when talking. This part is actually not necessary, but I wanted to show <code>plot_dist_comparison</code> because I find it really useful in practice </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Yes, this will be done when recording</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Love that, that sounds perfect!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>

Comment (AlexAndorra) [inline lesson_05.ipynb:72]: <p>Thanks! I'll be honest, it took me longer than I thought it would :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='116'/>

Comment (canyon289) [reply lesson_05.ipynb:72]: Theres one thats built into jupyter thats better to use now. Instructions on how to get it to work here

https://github.com/IntuitiveBayesInternal/IBCourseTemplate/tree/main/notebooks#notes-on-jupyter_contrib_nbextensions

Comment (AlexAndorra) [reply lesson_05.ipynb:72]: <p>Done</p>
